published/external/xdp/program.h

```diff
@@ -36,6 +36,8 @@ typedef enum _XDP_MATCH_TYPE {
     XDP_MATCH_TCP_QUIC_FLOW_DST_CID,
     XDP_MATCH_TCP_CONTROL_DST,
     XDP_MATCH_IP_NEXT_HEADER,
+    XDP_MATCH_INNER_IPV4_DST_MASK_UDP,
```

Comment:
We'll might need to add test cases for these to spinxsk.c, though since these conditions are so simple, I think we can rely on this:

https://github.com/microsoft/xdp-for-windows/blob/48117ff9c7322816c3393a2793c617c21f2837f3/test/spinxsk/spinxsk.c#L723-L725